core/http/src/main/java/org/trellisldp/http/WebAcFilter.java

```diff
@@ -88,6 +88,17 @@
     private final List<String> challenges;
     private final String baseUrl;
 
+    /**
+     * No-op for CDI
+     *
+     * @param accessService the access service
+     */
+    WebAcFilter() {
+    	this.accessService = null;
```

Comment:
Ditto.

Reply:
This is also fine, but you'll need to adjust the javadoc above, removing the `@param accessService`

Reply:
Same as above-- I don't think it's okay to leave actual broken code in place if "no one can see it". Why couldn't we use a `ServiceLocator`-based approach as we do elsewhere?